src/platform/forms-system/src/js/components/FieldTemplate.jsx

```diff
@@ -47,18 +47,19 @@ export default function FieldTemplate(props) {
   let errorSpanId;
   let errorSpan;
   let errorClass;
-  let errorStyle = 'none';
+  // let errorStyle = 'none';
   if (hasErrors) {
     errorClass = isDateField ? 'input-error-date' : 'usa-input-error';
     errorSpanId = `${id}-error-message`;
-    errorStyle = 'block';
+    // errorStyle = 'block';
     errorSpan = (
       <span className="usa-input-error-message" role="alert" id={errorSpanId}>
         <span className="sr-only">Error</span> {rawErrors[0]}
       </span>
     );
   }
 
+  /*
```

Comment:
we're going to gate this another way so that ti doesn't interfere with staging testing. commenting out for now to unblock other people.